cmd/helm/init.go

```diff
@@ -79,6 +79,7 @@ type initCmd struct {
 	opts           installer.Options
 	kubeClient     kubernetes.Interface
 	serviceAccount string
+	maxHistory     int
```

Comment:
- [ ] Did you consider using a `uint` here instead, for clarity?  
  Since zero is already a sentinel value, we could preclude negative numbers. However, if you wind up using this to create or index into slices, using a `uint` creates weird situations where you might be truncating values larger than the maximum value that `int` can hold.
- [ ] Consider naming this something more specific like "maxHistoryCount."  
  If we also wanted to be able to limit history by _size_ (byte count), then we'd wind up with one field called "maxHistory" and another something like "maxHistorySize" or "maxHistoryBytes." If you can accept that threat of what may never come, then your choice here is fine.